ntbs-service/Pages/Notifications/Edit/_SpecimenDetails.cshtml

```diff
@@ -66,33 +62,33 @@
                 </div>
                 
                 <br>
-                <h3>Patient Details</h3>
+                <h5 class="section-header-with-background">Patient Details</h5>

```

Comment:
And respectively `h6`?

Reply:
I thought h6 will be too small, but doesn't look bad. So changing it to h6 and the other one to h5

Reply:
It might also be worth decoupling visuals and semantics.
See https://beta.nhs.uk/service-manual/styles-components-patterns/typography#headings - you can apply different heading style sizes, whilst keeping the structure of the document semantically correct which helps non-visual users